content/sql/glossaries/operators/like.md

```diff
@@ -0,0 +1,120 @@
+---
+Title: "SQL Operators: `LIKE`"
+Subjects:
+  - "Language Fluency"
+  - "Code Foundations"
+  - "Data Science"
+Tags:
+  - "Database"
+  - "Queries"
+  - "Tables"
+Catalog Content:
+  - "https://www.codecademy.com/learn/paths/analyze-data-with-sql"
+  - "https://www.codecademy.com/learn/paths/design-databases-with-postgresql"
+---
+
+## Definition
+***
+Allows the user to perform case-sensitive pattern matching on `string` data through the use of wildcard characters.
+
+## Syntax
+***
+SELECT column1, column2, ...
+FROM table_name
+WHERE column LIKE 'pattern%';
+
+*OR*
+
+SELECT column1, column2, ...
+FROM table_name
+WHERE column LIKE '%pattern%'
+
+*OR*
+
+SELECT column1, column2, ...
+FROM table_name
+WHERE column LIKE '%pattern'
+
+*OR*
+
+SELECT column1, column2, ...
+FROM table_name
+WHERE column LIKE 'pattern_';
+
+*OR*
+
+SELECT column1, column2, ...
+FROM table_name
+WHERE column LIKE '_pattern_';
+
+*OR*
+
+SELECT column1, column2, ...
+FROM table_name
+WHERE column LIKE '_pattern%';
+
+*OR*
+
+SELECT column1, column2, ...
+FROM table_name
+WHERE column LIKE 'pattern__%';
+
+*OR*
+
+SELECT column1, column2, ...
+FROM table_name
+WHERE column LIKE 'pattern1%pattern2';
+
+## Example 1
+***
+Use `LIKE` as a `WHERE` condition to pattern match all sequences of characters in the `item_name` column from the `inventory` folder that begin with `abc`:
+```codebyte/sql
```

Comment:
Change ALL codebyte/sql** to just sql.

SQL is not one of the languages where we can use Codebytes. Make sure to really read the documentation closely: https://github.com/Codecademy/codepedia-content/blob/main/documentation/content-standards.md